src/content/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management.mdx

```diff
@@ -783,13 +819,16 @@ Items in this section could change as we work to bring our service provider into
 
 ## Next steps when you're done [#next-steps]
 
-### Downgrade users 
+### Downgrade users [#downgrade]
+
+When your users are provisioned in New Relic, you should then be able to see them in the [**User management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where). 
```

Comment:
I'm leery of "should".... Are there times when this won't be true? What should someone do if they don't see users in the User management UI?